CHANGELOG.md

```diff
@@ -24,6 +24,14 @@
 
 * [8988](https://github.com/grafana/loki/pull/8988) **darxriggs**: Promtail: Prevent logging errors on normal shutdown.
 
+#### Mixins
+
+#### Enhancements
+
+#### Fixes
+
+* [8995](https://github.com/grafana/loki/pull/8995) **dannykopping**: Mixins: check that `memcached_index_writes` exists before using it in `boltdb_shipper`
```

Comment:
What user facing error/bug does this change fix?

Reply:
You will get

```
evaluating jsonnet: RUNTIME ERROR: Field does not exist: memcached_index_writes
```

if you try and use the mixins with `use_boltdb_shipper: false`.

Reply:
I'd phrase the changelog entry accordingly then:

```suggestion
* [8995](https://github.com/grafana/loki/pull/8995) **dannykopping**: Mixins: Fix Jsonnet `RUNTIME ERROR` that occurs when you try to use the mixins with `use_boltdb_shipper: false`.
```

Reply:
Good call, thanks!